src/Compilers/VisualBasic/Portable/Errors/MessageProvider.vb

```diff
@@ -230,6 +230,12 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End Get
         End Property
 
+        Public Overrides ReadOnly Property WRN_AnalyzerReferencesFramework As Integer
+            Get
+                Throw ExceptionUtilities.Unreachable
+            End Get
+        End Property
```

Comment:
don't love that this is unreachable.  can we still have some sort of test for this (so that if we do add SG support for VB, this fires for .net framework as well)?